js/packages/openinference-core/src/utils/index.ts

```diff
@@ -25,6 +25,10 @@ export function withSafety<T extends GenericFunction>({
   };
 }
 
+const unknownJSONParse = (...args: Parameters<typeof JSON.parse>): unknown => {
+  return JSON.parse(...args);
```

Comment:
I think this is fine, but in general if we want to make it safe across all calls we can just change the type

```typescript
declare global {
  interface JSON {
    parse(text: string, reviver?: (this: any, key: string, value: any) => any): unknown;
  }
}

```